sdk/src/client/api/wait_for_tx_acceptance.rs

```diff
@@ -0,0 +1,48 @@
+// Copyright 2024 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use std::time::Duration;
+
+use crate::{
+    client::{Client, Error},
+    types::{api::core::TransactionState, block::payload::signed_transaction::TransactionId},
+};
+
+pub(crate) const DEFAULT_WAIT_FOR_TX_ACCEPTANCE_INTERVAL: Duration = Duration::from_millis(500);
+pub(crate) const DEFAULT_WAIT_FOR_TX_ACCEPTANCE_MAX_ATTEMPTS: u64 = 80;
+
+impl Client {
+    /// Checks the transaction state for a provided transaction id until it's accepted. Interval in milliseconds.
+    /// Returns the block id that contains this transaction.
```

Comment:
```suggestion
```

Reply:
Also needs to be removed from the wallet fn

Reply:
Done